cli/app.go

```diff
@@ -520,6 +520,155 @@ var app = &cli.App{
 				},
 			},
 		},
+		{
+			Name:      "train",
+			Usage:     "train on data",
+			UsageText: "viam data train [other options]",
```

Comment:
I think this should be just `viam train [other options]` now

Reply:
👍